src/components/priceAndTodaysPriceChange/index.js

```diff
@@ -0,0 +1,49 @@
+import { connect } from 'react-redux';
+
+import PriceAndTodaysPriceChange from './PriceAndTodaysPriceChange';
+import { addCommas } from '../../utils/formatting/dataItem';
+
+import {
+  LABEL_CURRENT_PRICE,
+  LABEL_OPEN,
+  VALUE_PRECISION_CURRENT_PRICE,
+  VALUE_PRECISION_PRICE_CHANGE,
+  VALUE_PRECISION_PRICE_CHANGE_PERCENTAGE
+} from '../../constants';
+
+const getStockData = state => {
+  const symbol = state.selectedStock;
+  const selectedStock = state.stocks[symbol];
+  return selectedStock.stockData;
+};
+
+const getCurrentPrice = stockData => {
+  const currentPrice = stockData[LABEL_CURRENT_PRICE].value;
+  return currentPrice.toFixed(VALUE_PRECISION_CURRENT_PRICE);
+};
+
+const calculatePriceChange = stockData => {
+  const openPrice = stockData[LABEL_OPEN].value;
+  return (getCurrentPrice(stockData) - openPrice)
+    .toFixed(VALUE_PRECISION_PRICE_CHANGE);
+};
+
+const calculatePriceChangePercentage = stockData => {
```

Comment:
bit of a misnomer, since it both 1) calculates and 2) formats result as a string